alacritty_terminal/src/event.rs

```diff
@@ -14,6 +14,9 @@ pub enum Event {
     Wakeup,
     Bell,
     Exit,
+    CursorStartBlinking,
+    CursorBlink,
```

Comment:
I don't think this is a good name, since `Blink` has no clear indication of what it should do and thus would make it difficult to implement. It should instead be something like `ToggleCursorVisibility`.

Reply:
Sure, that sounds like a good idea.